### DISCUSSION TITLE
Problema na execução da Calculadora

### DISCUSSION THREAD

**Discussion by ycaleb** [balta-io/2801#110] [Perguntas e Respostas] [answered]
Bom dia, fazendo a calculadora, na parte de soma deu tudo certo, porem, na de subtração, quando executo o programa com o dotnet run, o programa continua somando ao invés de subtrair...

![calc](https://user-images.githubusercontent.com/111933424/188474833-b252dac9-0a57-42c0-a644-5b2b4e89cacb.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @ycaleb, aparentemente você não salvou o arquivo (A bolinha branca no topo da tela indica isto). Desta forma o código sendo executado é o anterior.
